perf.go

```diff
@@ -120,7 +133,11 @@ func (m *PerfMap) Start() error {
 				}
 				continue
 			}
-			m.DataHandler(record.CPU, record.RawSample, m, m.manager)
+			if m.RecordHandler != nil {
+				m.RecordHandler(recordPtr, m, m.manager)
+			} else if m.DataHandler != nil {
+				m.DataHandler(recordPtr.CPU, recordPtr.RawSample, m, m.manager)
```

Comment:
Should we copy here, for the reason explained here: https://github.com/DataDog/ebpf-manager/pull/48/files#r895692199 ?

Reply:
Otherwise, it could break usages of this library if they upgrade without any code changes, since the `RawSample` is no longer a copy each time.

Reply:
A new object is allocated line 118